examples/json-csv/json_csv.bal

```diff
@@ -0,0 +1,73 @@
+import ballerina/io;
+import ballerina/log;
+
+// Converts a json into an array of strings
+// Returns the result as a tuple which contains the headers and fields
+function getFields(json rec) returns (string[], string[]) {
+    int count = 0;
+    string [] headers = [];
+    string [] fields = [];
+    headers = rec.getKeys();
+    foreach var field in headers {
+        fields[count] = rec[field].toString();
+        count = count + 1;
+    }
+    return (headers, fields);
+}
+
+// Example function which writes json content to csv
+function writeCsv(json content, string path) returns error? {
+    io:WritableCSVChannel csvch = io:openWritableCsvFile(path);
+    int recIndex = 0;
+    int recLen = content.length();
+    while (recIndex < recLen) {
+        (string [], string []) result = getFields(content[recIndex]);
+        var (headers, fields) = result;
+        if (recIndex == 0) {
+            //We ignore the result as this would mean a nill return
+             _  = check csvch.write(headers);
+        }
+        _  = check csvch.write(fields);
+        recIndex = recIndex + 1;
+    }
+    return;
+}
+
+public function main() {
+    // Sample json which will be written
+    json sample = {
+            "employees": {
+                "employee": [
+                    {
+                        "id": "1",
+                        "firstName": "Tom",
+                        "lastName": "Cruise",
+                        "photo": "https://ballerina-team/profile/3737.jpg"
+                    },
+                    {
+                        "id": "2",
+                        "firstName": "Maria",
+                        "lastName": "Sharapova",
+                        "photo": "https://ballerina-team/profile/5676.jpg"
+                    },
+                    {
+                        "id": "3",
+                        "firstName": "James",
+                        "lastName": "Bond",
+                        "photo": "https://ballerina-team/profile/6776.jpg"
+                    }
+                ]
+            }};
+    // Writes json into a csv
+    string path = "./files/sample.csv";
+    // Specify the json array which should be transformed into csv
+    // Also provide the location the csv should be written
+    var result = writeCsv(sample.employees.employee, path);
```

Comment:
Shall we rephrase the comments to say something along the lines of
```
// Invoke the function to write the `JSON` content to CSV, specifying the `JSON` array to write and the location to write to.
```

WDYT?

Reply:
In this case wouldn't invocation of the function be obvious ? restating it might not make sense i presume, intention was to highlight the type of params which should be provided when calling the function

Reply:
Yeah, felt a bit disconnected though. Shall we add periods at the end of the comments if we are keeping as is?